### DISCUSSION TITLE
Merge cards

### DISCUSSION THREAD

**Discussion by dobladov** [dobladov/youtube2Anki#23] [Ideas]
Some cards might be too short given once sentence, combining provides a fast way to reduce the number of exported cards.

![merge cards](https://user-images.githubusercontent.com/1938043/184131011-9217130a-ea1f-49ed-ba33-af44fbacc308.svg)

I do all these features on my vacations and free time, if someone really wants it soon, please consider [donating](https://www.paypal.com/donate/?hosted_button_id=Z4D6849QVUXD2).

**Comment by dobladov** [comment]
Implemented now 

https://user-images.githubusercontent.com/1938043/184427617-fd4ac43f-c06b-4290-bee2-a745343d47bc.mov

**Reply by Gardengul** [reply]
Automatic merging of segments according to criteria.
Here is an example of how this is implemented in similar applications.
![image](https://github.com/dobladov/youtube2Anki/assets/95347489/1e447707-445a-4c43-ae23-3bd3aa85bff5)
https://ankiweb.net/shared/info/939347702
